app/Services/GroupServiceProvider.js

```diff
@@ -49,7 +49,7 @@ async function destroyGroup(userId, groupId) {
   });
 
   if (counter === 0) {
-    throw new ApiError(httpStatus.NOT_FOUND, 'Group not found');
+    throw new ApiError(httpStatus.NOT_FOUND, 'Group not found', 'group');
```

Comment:
also in such cases. It only makes sense, if it is a form with multiple inputs that can throw errors independent.

Reply:
done